projects/pytorch/Dockerfile

```diff
@@ -16,18 +16,17 @@
 
 FROM sweetvishnya/ubuntu20.04-sydr-fuzz
 
-MAINTAINER Kirill Parhomenko
+MAINTAINER Theodor Arsenij Larionov-Trichkine
```

Comment:
typo - no e on end

Reply:
As far as I know, there are two spellings of my name. The one with an "e" at the end and the one without. Both should be appropriate :)